react/features/prejoin/components/preview/ParticipantName.js

```diff
@@ -85,17 +97,23 @@ class ParticipantName extends Component<Props> {
      */
     render() {
         const { value, isEditable, t } = this.props;
-        const { _onKeyDown, _onNameChange } = this;
+        const { _onKeyDown, _onNameChange, _onClearInput } = this;
```

Comment:
not shure if it's a good approach but you also can write
```suggestion
        const { 
            props: {
                value,
                isEditable,
                t
            },
            _onKeyDown,
            _onNameChange,
            _onClearInput
        } = this;
```